irr/src/OpenGL/Driver.cpp

```diff
@@ -1068,6 +1023,57 @@ void COpenGL3DriverBase::drawElements(GLenum primitiveType, const VertexType &ve
 	endDraw(vertexType);
 }
 
+void COpenGL3DriverBase::renderArray(const void *vertices, const void *indexList,
```

Comment:
not sure but maybe `drawGeneric` is a better name

Reply:
I did name this function according to the similar function in the legacy driver. Can rename, though.

Reply:
the reason I suggested a different name is the similarity with `glDrawArrays` while it actually mostly calls `glDrawElements`